openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py

```diff
@@ -279,19 +213,8 @@ def filings(
     return OBBject(results=Query(**locals()).execute())
 
 
-@router.command(model="ShareStatistics")
-def shrs(
-    cc: CommandContext,
-    provider_choices: ProviderChoices,
-    standard_params: StandardParams,
-    extra_params: ExtraParams,
-) -> OBBject[BaseModel]:
-    """Share Statistics. Share statistics for a given company."""
-    return OBBject(results=Query(**locals()).execute())
-
-
 @router.command(model="HistoricalStockSplits")
-def split(
+def splits(
```

Comment:
same -- what is different from above?

Reply:
There are `historical_splits`, by ticker, and then there should be a different calendar endpoint that does not go in the FA router.  The function `compsplit()` should be taken out of FA.